sys/suit/handlers_command_seq.c

```diff
@@ -375,7 +375,9 @@ static int _dtv_verify_image_match(suit_manifest_t *manifest, int key,
     if (res != 0) {
         return SUIT_ERR_COND;
     }
-    riotboot_flashwrite_finish(manifest->writer);
+    if (!IS_ACTIVE(MODULE_SUIT_TRANSPORT_MOCK)) {
+        riotboot_flashwrite_finish(manifest->writer);
+    }
```

Comment:
Maybe a comment would be as useful as the commit message here? (your call)

Reply:
Squashed in a comment!